imap_processing/ccsds/ccsds_data.py

```diff
@@ -42,10 +42,7 @@ class CcsdsData:
     def __init__(self, packet_header: dict):
         attributes = [field.name for field in fields(self)]
 
-        for key, item in packet_header.items():
-            value = (
-                item.derived_value if item.derived_value is not None else item.raw_value
-            )
+        for key, value in packet_header.items():
             if key in attributes:
```

Comment:
I think we still need to get `value.raw_value` here... won't value be of type `packets.IntParameter` instead of `int`?

Reply:
If I understood the original if statement, it seems like we want to get derived value if it's available or else get raw value. Now with new changes, we get derived value if it's available or else we get raw value. so getting the `value` will do what the if statement was doing before, if I understood correctly. That's how I understood at least when Gavin explain in the Slack thread. should we double check to be safe?

Reply:
If you could just print out the type and validate that it's a python type, that's good enough for me

Reply:
`packets.IntParameter` is a subclass of `int`

Reply:
in this case, are we good with current change?

Reply:
Well, if it's a different type, that's going to mess with some code, right? Some of my code depends on the types being accurate because of MyPy

Reply:
I guess, if you make the change and all the tests pass, then it's probably fine... but I'm just worried about changing all the types out from underneath everyone like this, even if they're basically the same as the original types. But perhaps that is just Python offending my delicate, type-loving sensibilities again

Reply:
Sure, then I think you can change the types to `IntParameter` now. But note that `raw_value` is not the same as `derived_value` which was being used before if it was present.